source/content/guides/decoupled/drupal-backend-starters/03-caching.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Decoupled Drupal Backend Starter for Front-End Sites
```

Comment:
@CobyPear It still looks like there is inconsistency as I click through all the pages in this folder: https://pr-8713-documentation.appa.pantheon.site/guides/decoupled/drupal-backend-starters/build-hooks

Just for consistency with the other pages in this folder it should be `Drupal Backend Starter for Front-End Sites` right?

Reply:
Yes, good catch. I thought for some reason it only said "Drupal Starter" for the other pages, apparently not.

Reply:
> I thought for some reason it only said "Drupal Starter" for the other pages, apparently not.

Probably because of these sections that are specific to the combos with Next.js and don't contain `Backend` in this title metadata

https://docs.pantheon.io/guides/decoupled/wp-nextjs-frontend-starters
https://docs.pantheon.io/guides/decoupled/drupal-nextjs-frontend-starters

So there's more opportunity for consistency. But I'll take it one step at a time.